extensions/wikia/CommunityPage/scripts/CommunityPageBenefitsPageviews.js

```diff
@@ -0,0 +1,37 @@
+require([
+	'wikia.pageviewsInSession',
+	'CommunityPageBenefitsModal',
+	'wikia.tracker',
+	'wikia.cookies'
+], function (pageviews, modal, tracker, cookies) {
+	'use strict';
+
+	var wikiaDomain = mw.config.get('wgDevelEnvironment') ? '.wikia-dev.com' : '.wikia.com',
+		// This cookie is checked in CommunityPageSpecialHooks::onBeforePageDisplay to avoid unnecessary script loading
+		modalShownCookieName = 'cpBenefitsModalShown',
+		modalShownExpirationTime = 2592000; // 30 days
```

Comment:
Are any of these variables going to be used outside of `setModalShownCookie`, or can we limit their scope to that method?